packages/node-sdk/README.md

```diff
@@ -138,6 +132,75 @@ The precedence for configuration options is as follows, listed in the order of i
 2. environment variable
 3. the config file
 
+## Type safe feature flags
+
+To get type checked feature flags, add the list of flags to your `bucket.ts` file.
+Any feature look ups will now be checked against the list you maintain.
+
+```typescript
+// Extending the Features interface to define the available features
+declare module "@bucketco/node-sdk" {
+  interface Features {
+    "show-todos": boolean;
+    "create-todos": boolean;
+    "delete-todos": boolean;
+  }
+}
+```
+
+![Type check failed](docs/type-check-failed.png "Type check failed")
+
+## Using with Express
+
+A popular way to integrate the Bucket Node.js SDK is through an express middleware.
+
+```typescript
+import bucket from "./bucket";
+import express from "express";
+import { BoundBucketClient } from "../src";
+
+// Augment the Express types to include the `boundBucketClient` property on the `res.locals` object
+// This will allow us to access the BucketClient instance in our route handlers
+// without having to pass it around manually
+declare global {
+  namespace Express {
+    interface Locals {
+      boundBucketClient: BoundBucketClient;
+    }
+  }
+}
+
+// Add express middleware
+app.use((req, res, next) => {
+  // Extract the user and company IDs from the request headers
+  // You'll want to use a proper authentication and identification
+  // mechanism in a real-world application
+  const { user, company } = extractBucketContextFromHeader(req);
```

Comment:
This function is referenced/defined anywhere in the README?
As a reader, I would try to find this function (is it imported? is the defined later in the code example?) which is frustrating/confusing. Can we show an actual working code example here?